### ISSUE TITLE
Show more rows of previous months

### ISSUE THREAD

**Issue by thomasglaser** [sfsam/Itsycal#204]
First of all: thanks, great app! Scrolling through months is a feature the mac didnt give me in such a handy way. But:

i would like to see (customize) more rows of the previous month or months. By clicking on the icon in the menu bar, the shown calendar can be half the screen height showing multiple previous and upcoming months while highlighting the current.

**Comment by sfsam** [comment]
Can you clarify a bit what you are looking for? Is it different from this?... https://www.mowglii.com/itsycal/help.html#extra_rows

**Comment by thomasglaser** [comment]
Yes, of course. Expanding to upcoming months is a nice feature - id like to see e. g. the whole previous month.

**Comment by sfsam** [comment]
I'm not sure I understand. Can't you see the whole previous month (as well as the current month) by going back 1 month?
